metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/HiveConnectorFastTableService.java

```diff
@@ -74,12 +74,12 @@ public HiveConnectorFastTableService(
         final HiveConnectorDatabaseService hiveConnectorDatabaseService,
         final HiveConnectorInfoConverter hiveMetacatConverters,
         final ConnectorContext connectorContext,
-        final DataSource dataSource,
+        @Qualifier("hiveJdbcTemplate") final JdbcTemplate jdbcTemplate,
```

Comment:
I might be missing something but the `@Autowired` here is kind of useless so is the `@Qualifier` cause the class isn't a classpath scanned bean...?